components/switch/gpio.rst

```diff
@@ -59,7 +59,7 @@ To create an active-low switch (one that is turned off by default), use the :ref
       - platform: gpio
         pin:
           number: 25
-          inverted: yes
+          inverted: True
```

Comment:
This change actually makes no difference, as in YAML both are equal in the YAML 1.1 specifications used.

Correct styling would be `true`/`false` in lower caps.


```suggestion
          inverted: true
```